lib/collections/schemas/accounts.js

```diff
@@ -7,6 +7,17 @@ import { Metafield } from "./metafield";
  * Accounts Schemas
  */
 
+const TaxSettings = new SimpleSchema({
+  exemptionNo: {
```

Comment:
I wonder if we shouldn't use the "switchable" option for schemas like we do for the product schema that could switch based on provider.